pkg/sqlqueries/SQL/cedar_bookmarked_systems/select_LOADER.sql

```diff
@@ -1,13 +1,3 @@
-WITH queried_cedar_system_ids AS (
-    SELECT cedar_system_id AS cedar_system_id, eua_user_id AS eua_user_id
-    FROM JSON_TO_RECORDSET(:param_table_json)
-        AS x("cedar_system_id" TEXT, "eua_user_id" TEXT) --noqa
-)
-SELECT
-   csb.cedar_system_id,
-   csb.eua_user_id,
-   csb.created_at
-FROM queried_cedar_system_ids qcsi
-INNER JOIN cedar_system_bookmarks csb 
-    ON csb.cedar_system_id = qcsi.cedar_system_id
-    WHERE csb.eua_user_id = qcsi.eua_user_id;
+SELECT eua_user_id, cedar_system_id
+FROM cedar_system_bookmarks
+WHERE (eua_user_id, cedar_system_id) = ANY(SELECT UNNEST($1::TEXT[]), UNNEST($2::TEXT[]));
```

Comment:
I don't think this is keeps the row level specificity that was provided by the serialization paradigm.
for this input
```
$1 = ["steven", "sam", "clay"]
$2 = ["wade", "lawson", "benson"]
```

I'd expect this output --> 
```
steven, wade
steven, lawson
steven, benson
sam, wade
sam, lawson
sam, benson
clay, wade
clay, lawson
clay, benson

```

I think the desired result set would be

```
steven, wade
sam, lawson
clay, benson
```

There are some ways you can UNNEST with ORDINALITY and join on order to get a matched set with this approach. I suspect `JSON_TO_RECORDSET` might be simpler in order to main key to value pairings.

Reply:
per conversation, will keep unnest but using `CAST()` with named params instead! Thanks Steven!

Reply:
Thanks Sam!
To call out for anyone taking a look, the way you are using unnest doesn't return a cartesian product like I was anticipating, so this works great! And if it is desired for a more complex situation, this could be put in a CTE. Thanks for syncing on this!